research-software-directory/RSD-as-a-service
# Improve scraper code

# Refactor scrapers code and add tests

Changes proposed in this pull request:

* Refactor scrapers code
* Add scrapers tests
* Run Maven JUnit tests on GitHub through workflow

How to test:

* Rebuild scrapers images
	* `docker-compose build scrapers && docker-compose up`
* If the build succeeds, then the unit tests passed
* Check that the programming languages, license and commit scrapers still work correctly
	

PR Checklist:
